src/main/java/org/spongepowered/api/util/NamedIdentifiable.java

```diff
@@ -22,5 +22,18 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-@org.spongepowered.api.util.annotation.NonnullByDefault
-package org.spongepowered.api.command.source;
+package org.spongepowered.api.util;
+
+/**
+ * An identifiable object that uses a {@link String} as an identifier.
+ */
+public interface NamedIdentifiable {
```

Comment:
`UniquelyNameable` or perhaps `UniquelyNamed`?

Reply:
Should just be `Named` or `Nameable` - existing `Nameable` interface should be renamed to something else.

Reply:
The existing one should become `Translatable`.

Reply:
Just had a look at what the existing `Nameable` is used for, it's used for `Inventory` only (and in the JDs for `QueryOperationTypes`. One thing we can do is:

* Rename `NamedIdentifiable` to `Nameable`
* Rename `Nameable` to `Translatable`
* Rename `Translatable#getName` to `Translatable#getTranslation`

We could also look to make player names `Translations`, but that's probably overkill.

Thoughts?

Reply:
I've been thinking about this more: the name `Translatable` for the current `Nameable` interface doesn't quite fit, nor does the proposed method rename. An inventory is not translatable, its name is, and that's what the interface indicates. So, I don't think `Translatable` is right here, nor is `getTranslation`.

This is the remaining thing left to decide before I merge this and I really don't have a good idea for what we should do - a name that indicates that it has a translatable name eludes me.

Reply:
[Okay, I've done this - folding the existing `Nameable` into the "new" `Nameable` as `Nameable.Translation`](https://github.com/SpongePowered/SpongeAPI/blob/1584092d1eb8fc8a578370ced2e6ce2d76254f91/src/main/java/org/spongepowered/api/util/Nameable.java). Provides the link, resolves the concerns.